scripts/split_eafs.py

```diff
@@ -56,9 +56,9 @@ def write_text(annotation, fname, ext):
     f.close()
```

Comment:
Generally prefer using context managers for opening files, this is currently being done in the rest of the file and hence would bring consistency benefits as well:

```python
with open(os.path.join(output_text_dir, fname + ext), 'w') as f:
     f.write(annotation)
```